core/common/src/IModel.ts

```diff
@@ -123,8 +127,8 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   autoUploadBlocks?: boolean;
 }
 
-/** Options that can be supplied when opening an existing StandaloneDb.
- * @beta
+/** Options supplied to open an existing StandaloneDb.

```

Comment:
StandaloneDb is internal. Change to "standalone BriefcaseConnection"?

Reply:
@kabentley I thought `StandaloneDb`s were no longer going to be used outside of tests. How would someone use them in a regular workflow?